test/integration/concept/.gitkeep

```diff
@@ -0,0 +1 @@
+Placeholder for upcoming integration tests - delete this file once the directory has other files in it
```

Comment:
Previously the directory was "kept alive" by an empty Java file. We felt that this was awkward and the right thing to do would be to use a .gitkeep instead.